packit_service/worker/events/event.py

```diff
@@ -392,6 +392,14 @@ def __init__(self, created_at: Optional[Union[int, float, str]] = None):
         self._db_project_object: Optional[AbstractProjectObjectDbType] = None
         self._db_project_event: Optional[ProjectEventModel] = None
 
+    @classmethod
+    @abstractmethod
+    def event_type(cls) -> str:
+        """
+        ‹TODO›
+        """
+        ...
```

Comment:
Have you tried to build it dynamically? I think something like the following should work?
```
cls.__module__ + "." + cls.__name__
```

Reply:
I've tried just now, I have 2 failing tests and I'm not really sure what's the cause of the issue

Reply:
Oh I see… that includes »whole« path… I'm not sure if it's worth it :thinking:

Reply:
We can always remove the unwanted prefix (or probably we need to use the new one everywhere?), so that we don't have to define the `event_type` for every subclass.